repo/profile/id.go

```diff
@@ -45,12 +45,31 @@ func (id *ID) UnmarshalYAML(unmarshal func(interface{}) error) (err error) {
 	return
 }
 
+// IDRawBytes constructs an ID from raw bytes. No decoding happens. Should only be used in tests
+func IDRawBytes(data string) ID {
```

Comment:
function name `RawBytes` that accepts a `string` type feels smelly to me. Can we rename to `IDRawByteString`?

Reply:
Will do, good idea.